### PR TITLE
 [docs] PR for Document Sequelize in Integrations (Doc9383) 

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

**Review by sanyamsinghal — CHANGES_REQUESTED** [review]
Suggested some changes. Other than that it LGTM

File: docs/content/latest/integrations/sequelize.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+menu:
+  latest:
+    identifier: sequelize
+    parent: integrations
+    weight: 650
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes how to use [Sequelize](https://sequelize.org/) with YugabyteDB to enable promise-based Node.js object-relational mapping (ORM).
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/sequelize.md:15]: Changed

File: docs/content/latest/integrations/sequelize.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  mkdir node_modules
+  touch demo.js
+  ```
+
+- A YugabyteDB-specific Sequelize package called  [sequelize-yugabytedb](https://www.npmjs.com/package/sequelize-yugabytedb). You can install this package by running the following command from the `demo_project` directory:
+
+  ```shell
+  npm install sequelize-yugabytedb
+  ```
+
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/sequelize.md:40]: Added

File: docs/content/latest/integrations/sequelize.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  SELECT * FROM CUSTOMERS;
+  ```
+
+The following illustration presents the output:
+
+![img](/images/ee/sequelize.png)
+
+The `createdAt` and `updatedAt` timestamp columns have been added automatically by Sequelize.
+
+## Limitations
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/sequelize.md:115]: Deleted section

**Review by sanyamsinghal — APPROVED** [review]
LGTM

**Pull request merged**